src/generator/operations.ts

```diff
@@ -651,7 +651,7 @@ function createProtocolRequest(group: OperationGroup, op: Operation, imports: Im
       text += `\turlPath = strings.ReplaceAll(urlPath, "{${pp.language.go!.serializedName}}", ${paramValue})\n`;
     }
   }
-  text += `\treq, err := runtime.NewRequest(ctx, http.Method${(<string>op.requests![0].protocol.http!.method).capitalize()}, ${hostParam})\n`;
+  text += `\treq, err := runtime.NewRequest(ctx, http.Method${capitalize(<string>op.requests![0].protocol.http!.method)}, ${hostParam})\n`;
```

Comment:
in other places the `<string>` gets pulled off, why is it necessary here? (for my own curiosity)

Reply:
Good catch, these casts are also now unnecessary, I've removed them.